app/src/main/java/com/sopt/gongbaek/presentation/ui/component/button/ImageSelector.kt

```diff
@@ -0,0 +1,122 @@
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sopt.gongbaek.R
+import com.sopt.gongbaek.presentation.type.SelectImageButtonType
+import com.sopt.gongbaek.presentation.util.extension.clickableWithoutRipple
+import com.sopt.gongbaek.ui.theme.GongBaekTheme
+
+@Composable
+fun ImageSelector(
+    selectImageButtonType: SelectImageButtonType,
+    modifier: Modifier = Modifier,
+    selectedIndex: Int? = null,
+    onIndexSelected: (Int) -> Unit = {}
+) {
+    val chunkedImages =
+        selectImageButtonType.imageButtonResIdList.chunked(selectImageButtonType.chunkedCount)
+    Column(
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        chunkedImages.forEachIndexed { rowIndex, imageRow ->
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp),
+                horizontalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                imageRow.forEachIndexed { columnIndex, imageResId ->
+                    val imageIndex = rowIndex * selectImageButtonType.chunkedCount + columnIndex
+                    Box(
+                        modifier = modifier
+                            .weight(1f)
+                            .clickableWithoutRipple { onIndexSelected(imageIndex) }
+                            .clip(RoundedCornerShape(4.dp))
+                    ) {
+                        Image(
+                            painter = painterResource(id = imageResId),
+                            contentDescription = null,
+                            modifier = Modifier.fillMaxSize(),
+                            contentScale = ContentScale.Crop
+                        )
+
+                        if (selectedIndex == imageIndex) {
+                            Box(
+                                modifier = Modifier
+                                    .fillMaxSize()
+                                    .background(
+                                        color = Color.Black.copy(alpha = 0.25f),
```

Comment:
[1]: 디자인 시스템 컬러 사용 부탁드립니다욤!🙇🏻‍♂️
```suggestion
                                        color = GongBaekTheme.colors.black.copy(alpha = 0.25f),
```